docs/reference/ml/apis/evaluate-dfanalytics.asciidoc

```diff
@@ -0,0 +1,100 @@
+[role="xpack"]
+[testenv="platinum"]
+[[evaluate-dataframe]]
+=== Evaluate {dfanalytics} API
+
+[subs="attributes"]
+++++
+<titleabbrev>Evaluate {dfanalytics}</titleabbrev>
+++++
+
+beta[]
+
+Evaluates the executed analysis on an index that is already annotated with a 
```

Comment:
I would like to suggest expanding a bit this introductory paragraph. I'll write up what I have in mind but then feel free to edit it in as you prefer.

After configuring an analysis and running it on a dataset we have the results of the analysis. At that point the question rises: how good are those results? We can answer this question by evaluating the quality of the results. Evaluation is typically done via calculating a set of metrics that capture various aspects of the quality of the results over data for which we have the `ground truth`. For different types of analyses different metrics are suitable. This API packages together commonly used metrics for various analyses.